distros/generate.sh

```diff
@@ -9,7 +9,13 @@ set -e
 #	or: ./generate.sh debian-newversion
 #		to create a new folder and a Dockerfile within it
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+# `readlink -f` does not work on macOS.
+# To get `greadlink`, run `brew install coreutils`.
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
How about I change the docs to make the dangers clear and provide two alternatives:
* Uninstall straight after,
* Temporarily create an alias / symlink
?

> If it's as trivial a change as you say

How about I make a PR and see what the reaction is. I would like this review to continue as if the PR wouldn't land, and get this merged if we are waiting on that DC/OS PR. If the DC/OS PR does land then I can take the `brew` install out of the README. In the meantime I can put a warning in the README here.